app/wyzebridge/wyze_api.py

```diff
@@ -10,14 +10,25 @@
 from typing import Any, Callable, Optional
 from urllib.parse import parse_qs, urlparse
 
+import urllib
+
```

Comment:
Random whitespace :slightly_smiling_face: 
I would go through and run `black --fast` if you haven't already. It should fix most of these kinds of issues.